styleguide/derek/_partials/_footer.jade

```diff
@@ -1,2 +1,2 @@
-.container-fluid.bg-blue-gray.standard-padding
+.container-fluid.bg-blue-gray.fluid-footer-container
```

Comment:
@bighappyface I included it like the navigation file includes the ceiling and mainnav partials. Since the footer wont ever appear without the rug and basement I felt like it would be easier to include them within that page as a group but have them in their own partials.